packages/charts/src/core/grid/radial-grid.ts

```diff
@@ -15,15 +16,19 @@ export class RadialGrid extends Grid implements IGrid {
     }
 
     public updateDimensions(dimensions: IDimensions): IGrid {
-        this.config().dimension
-            .outerWidth(dimensions.width)
-            .outerHeight(dimensions.height);
+        const dimensionConfig = this.config().dimension;
+        if (!isUndefined(dimensions.width)) {
+            dimensionConfig.outerWidth(dimensions.width)
+        }
+        if (!isUndefined(dimensions.height)) {
+            dimensionConfig.outerHeight(dimensions.height)
+        }
```

Comment:
There was a bug with the radial grid where you couldn't manually set the grid dimensions without a crash here.